zenoh/src/query.rs

```diff
@@ -294,6 +306,22 @@ impl<'a, 'b, Handler> GetBuilder<'a, 'b, Handler> {
         self
     }
 
+    #[zenoh_macros::unstable]
+    pub fn attachment(&self) -> Option<&Attachment> {
```

Comment:
Similar comment as per other builders. What's the interest of attachment and attachment_mut in the PutBuilder? Shouldn't with_attachment be enough? It's a builder, so I would expect to be additive for the attachment...